spec/lib/miq_automation_engine/methods/wait_for_ip_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+require Rails.root.join('db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/wait_for_ip').to_s
+require Rails.root.join('spec/support/miq_ae_mock_service').to_s
```

Comment:
@mkanoor I agree with @Fryguy that we can move these over with the other automate model tests.  This would keep it from running in the vmdb test suite.

@Fryguy - Note, however, that this is not testing the automate method like we were before.  (I.E. load the entire model and run with `MiqAeEngine.instantiate`)  This newer style allows us to test methods individual without requiring all the overhead.